sc/social/like/tiles/facebook.pt

```diff
@@ -0,0 +1,18 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="sc.social.like">
+  <body>
+    <div class="tile-facebook tile-content">
+      <p tal:condition="python:view.is_empty and view.is_compose_mode()">
+        To use this tile you must define a Facebook application ID in the Social Like configlet.
+      </p>
+      <iframe src="" width="340" height="500" style="border:none;overflow:hidden"
+          tal:condition="not:view/is_empty"
+          tal:attributes="src string://www.facebook.com/plugins/page.php?${view/get_data};
+                          width view/width;
+                          height view/height"></iframe>
```

Comment:
the Facebook documentation gives you 2 options: JavaScript SDK and `iframe` tag; I alway prefer the `iframe` when available:

http://stackoverflow.com/a/5359885